rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -85,3 +85,6 @@ nova_ceilometer_enabled: False
 swift_ceilometer_enabled: False
 keystone_ceilometer_enabled: False
 tempest_service_available_ceilometer: False
+
+#Enable noVNC over spice console
```

Comment:
# Enable noVNC instead of Spice

space after the # (formatting thing) and clearer wording